enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Editor.tsx

```diff
@@ -78,7 +73,6 @@ export const Editor = ({
           }}
           setQueryBuilderMode={() => {}}
           hasVisualizeButton={true}
-          modelsFilterList={modelsFilterList}
```

Comment:
this prop is used for the modern data picker we use for the internal use case. We don't use that in embedding anymore since https://github.com/metabase/metabase/pull/50535. 

This is the modern data picker I mentioned.
![image](https://github.com/user-attachments/assets/7ebb2599-0981-4b87-a5b4-47ad6f6cf9d8)